starlette/responses.py

```diff
@@ -207,11 +207,13 @@ def __init__(
         background: BackgroundTask = None,
         filename: str = None,
         stat_result: os.stat_result = None,
+        send_header_only: bool = False,
```

Comment:
Let's use `method: str = None` here. It'll be marginally nicer if the developer can just do `method=request.method`.